test/features/prepare_config.feature

```diff
@@ -0,0 +1,25 @@
+# Date:  08/07/2020
+# Author: Eugeniu Costetchi
+# Email: costezki.eugen@gmail.com
+
+Feature: Prepare the skos-history config file and folder structure
```

Comment:
To my understanding, this feature should be done before this one `Running the skos-history diff of two dataset versions` right?

Reply:
precisely.